installer/PowerToysSetup/generateFileList.ps1

```diff
@@ -14,43 +14,14 @@ Param(
     # launcher plugins are being loaded into launcher process,
     # so there are some additional dependencies to skip
     [Parameter(Mandatory = $False, Position = 5)]
-    [bool]$isLauncherPlugin,
-    # Skip winAppSDK dlls as those are hard-linked
-    [Parameter(Mandatory = $False, Position = 6)]
-    [bool]$isWinAppSdkProj
+    [bool]$isLauncherPlugin
 )
 
 $fileWxs = Get-Content $wxsFilePath;
 
-#Skip PowerToysInterop files
-$coreWxs = Get-Content $PSScriptRoot/"Core.wxs"
-$coreWxs | ForEach-Object {
-    if ($_ -match "(<?define PowerToysInteropFiles=)(.*)\?>") {
-        [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', 'fileList',
-        Justification = 'variable is used in another scope')]
+$fileExclusionList = @("*.pdb", "*.lastcodeanalysissucceeded", "createdump.exe", "powertoys.exe")
 
-        $interopFilesList = $matches[2] -split ';'
-        return
-    }
-}
-
-#Skip WinAppSdk files
-if ($isWinAppSdkProj -eq $True) {
-    $winAppSDKWxs = Get-Content $PSScriptRoot/"WinAppSDK.wxs"
-    $winAppSDKWxs | ForEach-Object {
-        if ($_ -match "(<?define WinAppSDKFiles=)(.*)\?>") {
-            [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', 'fileList',
-            Justification = 'variable is used in another scope')]
-
-            $winAppSDKfilesList = $matches[2] -split ';'
-            return
-        }
-    }
-}
-
-$fileExclusionList = @("*Test*", "*.pdb", "*.lastcodeanalysissucceeded", "createdump.exe") + $interopFilesList + $winAppSDKfilesList
-
-$fileInclusionList = @("*.dll", "*.exe", "*.json", "*.msix", "*png", "*gif", "*ico", "*cur", "*svg", "index.html", "reg.js", "gitignore.js", "monacoSpecialLanguages.js", "resources.pri")
+$fileInclusionList = @("*.dll", "*.exe", "*.json", "*.msix", "*.png", "*.gif", "*.ico", "*.cur", "*.svg", "index.html", "reg.js", "gitignore.js", "monacoSpecialLanguages.js", "*.pri")
 
 $dllsToIgnore = @("System.CodeDom.dll", "WindowsBase.dll")
```

Comment:
They should be aligned... I don't think we're using the `$fileDepsJson` path anymore, except for PTRun plugins, so that shouldn't really do anything, right? 🤔

Reply:
Now that I look at it more closely that means there's more savings to be add ignoring PowerToys.Interop.dll 😄